dbms/src/Storages/DeltaMerge/tests/gtest_segment_read_task.cpp

```diff
@@ -269,6 +269,78 @@ class DMStoreForSegmentReadTaskTest : public DeltaMergeStoreTest
 
         ASSERT_INPUTSTREAM_BLOCKS(local_stream, remote_blks);
     }
+
+    static void checkDMFile(const DMFilePtr & dmfile_wn, const DMFilePtr & dmfile_cn)
```

Comment:
Declare as a static function of `DMStoreForSegmentReadTaskTest`. Because `DMStoreForSegmentReadTaskTest` is friend class of `DMFile`, so it can access to the private members of `DMFIle`.